rpc/client/main_test.go

```diff
@@ -19,6 +19,8 @@ func TestMain(m *testing.M) {
 	}
 
 	app := kvstore.NewPersistentApplication(dir)
+	// If testing block event generation
+	// app.SetGenBlockEvents() needs to be called here
```

Comment:
@jmalicevic please review this one as well (manual conflict resolution), is this well-inserted here?

Reply:
Yes this is the place to have this!